packages/react-components/react-table/src/index.ts

```diff
@@ -97,3 +97,25 @@ export {
 } from './TablePrimaryCell';
 
 export type { TablePrimaryCellProps, TablePrimaryCellSlots, TablePrimaryCellState } from './TablePrimaryCell';
+
+export {
+  TableCellLayout,
+  useTableCellLayoutStyles_unstable,
+  useTableCellLayout_unstable,
+  renderTableCellLayout_unstable,
+  tableCellLayoutClassNames,
+} from './TableCellLayout';
+export type { TableCellLayoutProps, TableCellLayoutSlots, TableCellLayoutState } from './TableCellLayout';
+
+export {
+  TableCellPrimaryLayout,
+  tableCellPrimaryLayoutClassNames,
+  useTableCellPrimaryLayoutStyles_unstable,
+  useTableCellPrimaryLayout_unstable,
+  renderTableCellPrimaryLayout_unstable,
+} from './TableCellPrimaryLayout';
```

Comment:
TBH I think that we can get this in a single component, https://fluentsite.z22.web.core.windows.net/0.64.0/components/item-layout/definition

And may be it should not be connected to Table at all